docs/reference.md

```diff
@@ -263,6 +264,54 @@ if (Platform.OS === 'ios') {
   });
 }
 ```
+
+
+---
+### startBuyerVerificationFlow
+
+Displays the enabled buyer verification flow. The method takes two configuration object and three call back parameters which correspond
```

Comment:
Wondering if we should keep this descpription consistent with this doc https://github.com/square/in-app-payments-flutter-plugin/blob/master/doc/reference.md#startbuyerverificationflow?

Reply:
It's done